test/run-script-pkg.js

```diff
@@ -6,18 +6,17 @@ const isWindows = process.platform === 'win32'
 const emptyDir = t.testdir({})
 
 const pkill = process.kill
-const consoleLog = console.log
 
-const mockConsole = t => {
-  const logs = []
-  console.log = (...args) => logs.push(args)
-  t.teardown(() => console.log = consoleLog)
-  return logs
-}
+const output = []
+process.on('output', (level, ...args) => {
+  if (level === 'standard') {
+    output.push([...args])
+  }
+})
+t.afterEach(() => output.length = 0)
```

Comment:
We should remove the `output` event from `process` on teardown I think. Won't change anything currently but could prevent a footgun in future refactors.